containers/nginx-storage/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openresty/openresty:alpine-fat
+
+COPY install-dependencies.sh /tmp/
+RUN /tmp/install-dependencies.sh
+
+COPY table_length.lua        /usr/local/openresty/lualib/
+CMD ["/usr/local/openresty/bin/openresty", "-g", "daemon off;"]
```

Comment:
I don't find that option `-g ` have long name. Include `-g daemon off;` in dockerfile ensure that nginx stay in the foreground, otherwise container will stop immediately after starting

Reply:
Ok, I see that it's default option in openresty:alpine-fat image, so We don't need it, in our Dockerfile.